modules/configure-an-additional-clientCA.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/api-server.adoc
+
+:_content-type: PROCEDURE
+[id="configure-an-additional-clientCA-for-the-OpenShift-API-server_{context}"]
+
+= Configure an additional clientCA for the OpenShift API server
+
+Optionally, you may choose to invalidate the installer-generated kubeconfig. You would do this when:
+* You don't trust who installed the cluster
+* The kubeconfig is leaked
+* Other security-related needs exist, such as periodic rotation of the kubeconfig
+
+To replace the installer-generated kubeconfig, remove the installer-generated clientCA from the API server
+
+. Import an additional CA certificate in a configmap of the openshift-config namespace. The CA file must be in PEM format.
++
+[source,terminal]
+----
+oc create configmap client-ca-custom -n openshift-config --from-file=ca-bundle.crt=ca.crt
+----
++
+. Patch the APIServer instance.
++
+[source, terminal]
+----
+oc patch apiserver cluster --type=merge -p '{"spec": {"clientCA": {"name": "client-ca-custom"}}}'
+----
+
+. Test the new clientCA certificate with a certificate signed from the new clientCA.
+. If the test is successful, you can remove the installer-generated clientCA.
```

Comment:
You are not explaining how to remove the installer-generated client CA yet you ask to remove it.

Reply:
I'd remove this reference from this section but add a new section that explains the removal, after this section, and that suggests that you should first add the new CA as per the section above before replacing.

Pietro may provide a better view on how it should exactly look like

Reply:
Thanks for the info, Pablo. Could you or Pietro send me the details on how to remove the installer-generated client CA?

Reply:
@nalhadef here is a bit confusing, this KCS explains the procedure: https://access.redhat.com/solutions/6054981

To give you more context:
- the installer-generate kubeconfig contains a client TLS certificate signed by a CA
- the CA signing the kubeconfig is stored into the configmap named `admin-kubeconfig-client-ca` into the `openshift-config` namespace
- in order to invalidate the installer-generated kubeconfig you have to replace the CA (thus the content of the configmap)
- replacing the CA can be dangerous because if you are using the installer-generated kubeconfig and the content of the CM is somehow invalid you can be locked out from the API
- for that reason in the KCS I suggest to:
   - first add an additional CA
   - generate a new kubeconfig using the new CA
   - replace the original CA replacing the configmap.

Reply:
Incorporated KCS article content